lib/pulfalight/traject/ead2_config.rb

```diff
@@ -371,15 +373,21 @@
     segments = value.split
 
     filtered = segments.reject { |e| e =~ /^[[:punct:]]/ }
```

Comment:
This is very strange to me. I don't understand why we need to split on punctuation and figure out what's a language when the XML is already split out in the EAD. I wonder if we need to pull it from `/ead/archdesc/did/langmaterial/language` instead, so it doesn't combine all the languages?

Alternatively I think there's a way get the nodes out raw so we can look at just each individual language tag.

Can you spend a little time looking into that? I'm scared parsing this text out is gonna lead to strange bugs.